core/src/test/java/io/questdb/std/fastdouble/FastDoubleSwarTest.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * @(#)FastDoubleParserTest.java
+ * Copyright © 2022. Werner Randelshofer, Switzerland. MIT License.
+ */
+
+package io.questdb.std.fastdouble;
+
+/**
+ * Tests class {@link FastDoubleParser}
+ */
+public class FastDoubleSwarTest {
```

Comment:
This class seems to be unused.

Reply:
Would you mind removing them? There are three or four from the ordinal repository

Reply:
Addressed in 2ff64e43467de473b9a567492ee0d63ba6e9e81d